packages/CopyInput/src/CopyInput.styles.js

```diff
@@ -0,0 +1,19 @@
+import styled, { css } from "styled-components";
+import tokens from "@paprika/tokens";
+
+export const CopyInput = styled.div(
+  () => css`
```

Comment:
If you don't need any of the `props` for styling, you don't need the function here.  You can just do:
```js
export const CopyInput = styled.div`
```
Followed by your css.